digdag-storage-s3/src/main/java/io/digdag/storage/s3/S3Storage.java

```diff
@@ -197,8 +200,11 @@ public void list(String keyPrefix, FileListing callback)
     @Override
     public Optional<DirectDownloadHandle> getDirectDownloadHandle(String key)
     {
+        final long secondsToExpire = config.has("direct_download_expiration") ?
+                config.get("direct_download_expiration", Long.class) : 10*60;
```

Comment:
@muga 
I think you can use this method to set default value.

https://github.com/treasure-data/digdag/blob/188320b30bf2889315d33557d63a36e0ff79a20f/digdag-client/src/main/java/io/digdag/client/config/Config.java#L268-L275

Reply:
Thank you. Fixed 5d1295ca2c737cd32cebe72232c6dd2846bf4368